src/System.Runtime.Serialization.Formatters/tests/SerializationTypes.cs

```diff
@@ -78,23 +81,40 @@ public override bool Equals(object obj)
     }
 
     [Serializable]
-    public class Point
+    public class Point : IComparable<Point>, IEquatable<Point>
     {
         public int X;
         public int Y;
 
+        public Point()
```

Comment:
@krwq is following up on your question and testing locally without the default constructor. I think there was a reason why I had to add it.